src/Cli/dotnet/UILanguageOverride.cs

```diff
@@ -24,7 +26,20 @@ public static void Setup()
 
         private static void ApplyOverrideToCurrentProcess(CultureInfo language)
         {
+
+            CultureInfo.DefaultThreadCurrentCulture = language;
```

Comment:
>CultureInfo.DefaultThreadCurrentCulture = language; [](http://example.com/codeflow?start=12&length=51)

Why you are changing this?

Reply:
This is so the date formatting and such are aligned to the same culture as the UI Culture, https://stackoverflow.com/questions/329033/what-is-the-difference-between-currentculture-and-currentuiculture-properties-of

Reply:
It is not right to force the User and UI languages to be same. Imagine I am a user like to have English UI and use Italian language for the date/time formatting. This change is going to break that.

Reply:
I see, I'll remove this part. I didn't realize this was controlled by a flag/option in Windows